README.md

```diff
@@ -0,0 +1,22 @@
+# mkbtr
+
+Generate Switch NCA updates using BKTR tables.
+
+## Usage
+
+To use the main mkbktr generation tool:
+`$ mkbtr [original nca] [patched nca] [output nca] [/path/to/prod.keys]`
+
+TODO: add guide for small utils
+
+## Building
+
+Building is the standard CMake process:
+```
+mkdir build
+cd build
+ccmake ../
```

Comment:
Typo? Or does `ccmake` work?

Reply:
ccmake is the curses version of cmake that is friendly and prompts you for errors. I'm willing to swap it back to cmake, but I'm not great at remembering the cache commands so I just tend towards the interactive version.